### PR TITLE
Adding a page to the Guides about PyNE

### PR DESCRIPTION
**Pull request by Dayvis7**
## Summary of changes

This PR adds a page to the website that provides basic information on PyNE, as well as many links to various parts of the main PyNE website for installing and developing PyNE.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [x] I have read the **CONTRIBUTING** document.
- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

## Associated Issues and PRs

- Issue: #177

### CODE REVIEW

File: manual/guides/pyne.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+---
+layout: manual
+title: PyNE
+subtitle: "An important tool in our toolkit"
+permalink: /manual/guides/pyne
+---
+
+# <a name="a brief overview"></a>A Brief Overview
+
+Taken from the [PyNE website](http://pyne.io/index.html), PyNE is intended to be a library used for nuclear engineering. It can be used to help its users "be more productive, have the *best* solvers, have a beautiful API, write really great code, and teach the next generation". The official website provides all the information that one would need, including information about [installation](http://pyne.io/install/index.html) and a [tutorial](http://pyne.io/tutorial/index.html) on using PyNE.
```
Comment (samgdotson) [inline manual/guides/pyne.md:10]: 1. I'm not sure where the quote is from, I couldn't find it on the website. Is this a quote from @katyhuff ?
2. I'd recommend reviewing Katy's writing checklist, particularly about using active voice vs passive voice and "throat clearing."

File: manual/guides/pyne.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+
+# Development
+
+One of the perks of PyNE is that it is open source and free to use. Prof. Huff encourages contributions to PyNE, as its purpose is to provide a complete library meant to help all nuclear engineers. As such, anyone who feels that there should be something added or something changed is welcome to proceed in the proper manner to have their changed included in the final product, making their work easier. The website provides a [developer's guide](http://pyne.io/devsguide/index.html) intended to assist with testing, developing, and integration with PyNE. Prof. Huff is willing to help by offering some guidance on contributions, so any ideas on fixes or changes are encouraged.
```
Comment (samgdotson) [inline manual/guides/pyne.md:14]: 1. Once again, review the writing checklist.
2. This is a great place to link to the PyNE github.

File: manual/guides/pyne.md (hunk 1 above)
Comment (Dayvis7) [reply manual/guides/pyne.md:10]: I'll check out the writing and try to make it more active and reduce the filler words. But that quote was from the tutorial page: http://pyne.io/tutorial/00.html

Comment (Dayvis7) [reply manual/guides/pyne.md]: @samgdotson let me know what you think. I worked on it a bit more

File: manual/guides/pyne.md [hunk 3]
```diff
@@ -0,0 +1,14 @@
+---
+layout: manual
+title: PyNE
+subtitle: "An important tool in our toolkit"
+permalink: /manual/guides/pyne
+---
+
+# A Brief Overview
+
+PyNE is a library used for nuclear engineering, helping its users "be more productive, have the *best* solvers, have a beautiful API, write really great code, and teach the next generation". The [PyNE website](http://pyne.io/index.html) includes information about [installation](http://pyne.io/install/index.html) and a [tutorial](http://pyne.io/tutorial/index.html) on using PyNE.
```
Comment (katyhuff) [inline manual/guides/pyne.md:10]: Please hard wrap your lines (<80 chars per line).
```suggestion
PyNE is a library used for nuclear engineering, helping its users "be more productive, 
have the *best* solvers, have a beautiful API, write really great code, and teach the next 
generation". The [PyNE website](http://pyne.io/index.html) includes information about 
[installation](http://pyne.io/install/index.html) and a [tutorial](http://pyne.io/tutorial/index.html) 
on using PyNE.
```

File: manual/guides/pyne.md [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+
+# Development
+
+PyNE is open source and free to use. Prof. Huff encourages contributions to PyNE, as its purpose is to provide a complete library meant to help all nuclear engineers. Anyone who feels that there should be something added or something changed is encouraged to develop changes. The website provides a [developer's guide](http://pyne.io/devsguide/index.html) to assist with testing, developing, and integration with PyNE, and the [Pyne GitHub](https://github.com/pyne/pyne) contains the source code. Prof. Huff is willing to help by offering some guidance on contributions, so any ideas on fixes or changes are encouraged.
```
Comment (katyhuff) [inline manual/guides/pyne.md:14]:
```suggestion
PyNE is open source and free to use. Prof. Huff encourages contributions to PyNE, 
as its purpose is to provide a complete library meant to help all nuclear engineers. 
Anyone who feels that there should be something added or something changed is 
encouraged to develop changes. The website provides a 
[developer's guide](http://pyne.io/devsguide/index.html) to assist with testing, 
developing, and integration with PyNE. The [Pyne GitHub Organization](https://github.com/pyne/pyne) 
contains the source code. Prof. Huff is willing to help by offering some guidance 
on contributions, so any ideas on fixes or changes are encouraged.
```

**Comment by katyhuff** [comment]
(once @samgdotson  approves, this can be merged. please merge when ready, Sam.)

**Comment by samgdotson** [comment]
Looks great, thanks Gavin. 
Merging.

**Pull request merged**